### ISSUE TITLE
Conditionally show plugin features in Reactotron app based on client app config 

### ISSUE THREAD

**Issue by joshuayoes** [infinitered/reactotron#1345]
Currently in Reactotron, all of the features are available in the UI, such as image overlay, even if the client app does not have the necessary plugins set up.

Reactotron app should either grey out these features, suggest how to enable them, or conditionally hide them all together if the client app config isn't set up.

**Comment by frankcalise** [comment]
Yeah I was thinking about this as I was adding the Apollo Client tabs.

Does the desktop app already know what plugins are running and we just need to conditionally them? Or do we need to add a pattern for a plugin for it to announce its registration when `onConnect` hits?

**Comment by joshuayoes** [comment]
My thought is that we could add an `id` key to each plugin and then we could send it to Reactotron app in the `connect` call. 

This would be a breaking change but I'm not sure how else we'd get this infos
